apps/stripe/src/app/api/saleor/transaction-initialize-session/use-case.test.ts

```diff
@@ -92,19 +98,20 @@ describe("TransactionInitializeSessionUseCase", () => {
       appConfigRepo: mockedAppConfigRepo,
       stripePaymentIntentsApiFactory: testStripePaymentsIntentsApiFactory,
     });
+
     const responsePayload = await uc.execute({
       channelId: mockedSaleorChannelId,
       saleorApiUrl: mockedSaleorApiUrl,
       appId: mockedSaleorAppId,
       event: getMockedTransactionInitializeSessionEvent(),
     });
 
-    expect(responsePayload._unsafeUnwrapErr()).toBeInstanceOf(
-      TransactionInitializeSessionUseCase.UseCaseError,
+    expect(responsePayload._unsafeUnwrap()).toBeInstanceOf(
+      TransactionInitalizeSessionUseCaseResponses.ChargeFailure,
     );
   });
 
-  it("Throws error when currency coming from Saleor is not supported", async () => {
+  it("Returns BrokenAppResponse when currency coming from Saleor is not supported", async () => {
```

Comment:
Is that right? I think it should be "InvalidRequest". 
App will document supported currencies and if someone tried to pay for Dogecoin, it's not app that is broken but incoming call